framework/db/oci/QueryBuilder.php

```diff
@@ -49,7 +49,29 @@ class QueryBuilder extends \yii\db\QueryBuilder
     /**
      * @inheritdoc
      */
-    protected $likeEscapeCharacter = '\\';
+    protected $likeEscapeCharacter = '!';
+    /**
+     * @inheritdoc
+     */
+    protected $likeEscapingReplacements = [
+        '%' => '!%',
+        '_' => '!_',
+        '!' => '!!',
+        // \ will be checked later Because yii\db\Schema::quoteValue may quote it
```

Comment:
Suggestion: `\ is initialized in init() method since we need to choose replacement value based on [[\yii\db\Schema::quoteValue()]]`.